meta-edison-bsp/recipes-bsp/flashall/flashall-2.0/flashall.sh

```diff
@@ -158,6 +160,11 @@ while true; do
 	esac
 done
 
+if test -z ${IMAGE_NAME} ; then
+	echo "Image name not specified, please use -i / --image parameter!" ;
+	echo "" ;
```

Comment:
as expected argument is not full name of the file, would it make sense into error message or into usage line to add example of the expected argument ?